docs/CountMinSketch_Commands.md

```diff
@@ -1,21 +1,55 @@
-# RedisBloom Count Min Sketch Documentation
+# RedisBloom Count-Min Sketch Documentation
 
-## CMS.INITBYDIM
+## Create
 
-### Format:
+### CMS.INITBYDIM
 
-```
+Initializes a Count-Min Sketch to dimensions specified by user.
+
+```sql
 CMS.INITBYDIM key width depth
 ```
 
-Initializes a CMS to the dimensions specified by width and depth.
+### Parameters:
 
-## Parameters:
+* **key**: The name of the sketch.
+* **width**: Number of counter in each array. Reduces the error size.
+* **depth**: Number of counter-arrays. Reduces the probability for an
+    error of a certain size (percentage of total count).
+    
+### Complexity
+
+O(1)
+
+### Return
+
+OK on success, error otherwise
 
-* **key**: Key under which the sketch is to be found.
-* **width**: Number of counters kept in each array.
-* **Depth**: Number of arrays.
+#### Reserve example
```

Comment:
Not a `Reserve` command - you might just want to go with a generic `Example` header.